src/generic/course-upload-image/messages.js

```diff
@@ -0,0 +1,38 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  uploadImageHelpText: {
+    id: 'course-authoring.schedule-section.introducing.upload-image.help-text',
+    defaultMessage: 'Please provide a valid path and name to your {identifierFieldText} (Note: only JPEG or PNG format supported)',
+  },
+  uploadImageFilesAndUploads: {
+    id: 'course-authoring.schedule-section.introducing.upload-image.file-and-uploads',
+    defaultMessage: 'files and uploads',
+  },
+  uploadImageDropzoneText: {
+    id: 'course-authoring.schedule-section.introducing.upload-image.dropzone-text',
+    defaultMessage: 'Drag and drop your {identifierFieldText} here or click to upload.',
+  },
+  uploadImageDropzoneAlt: {
+    id: 'course-authoring.schedule-section.introducing.upload-image.dropzone-text',
```

Comment:
This identifier is a duplicate, please rename

Reply:
Good point, thanks! Changed